Makefile

```diff
@@ -1,5 +1,5 @@
-LINKER_FLAGS := $(shell pkg-config --libs gtk+-3.0)
-C_FLAGS := $(shell pkg-config --cflags gtk+-3.0)
+LINKER_FLAGS := $(shell pkg-config --libs gtk+-3.0 gdk-3.0 pangocairo pangoft2 pango gio-unix-2.0 glib-2.0)
```

Comment:
I'm not sure if all these packages are needed. I took inspiration from https://github.com/rhx/SwiftGtk